napari/utils/theme.py

```diff
@@ -222,12 +229,7 @@ def get_theme(name, as_dict=None):
         )
         as_dict = False
     if as_dict:
-        _theme = _theme.dict()
-        _theme = {
-            k: v if not isinstance(v, Color) else v.as_rgb()
-            for (k, v) in _theme.items()
-        }
-        return _theme
+        return _theme.dict()
```

Comment:
I know that it is not entirely in this scope, but in the context of a variable in a function, the variable starting from `_` is meant to be unused. So `_theme` should be rather `theme_`